BO4E/BO/Messlokation.cs

```diff
@@ -248,14 +274,40 @@ public class Messlokation : BusinessObject
 
         /// <summary>
         ///   Zugeordnete Messprodukte
-        /// </summary>        
+        /// </summary>
         [JsonProperty(Required = Required.Default, Order = 30, PropertyName = "messprodukte")]
         [JsonPropertyName("messprodukte")]
         [ProtoMember(1024)]
         [JsonPropertyOrder(30)]
         [NonOfficial(NonOfficialCategory.CUSTOMER_REQUIREMENTS)]
         public List<Messprodukt>? Messprodukte { get; set; }
 
+        /// <summary>
+        /// Lokationszuordnung, um bspw. die zugehörigen Messlokationen anzugeben
```

Comment:
Nur, weil wir hier schon in einer Messlokation sind ;)
```suggestion
        /// Lokationszuordnung, um bspw. die zugehörigen Marktlokationen anzugeben
```